testing/file_test/rules.bzl

```diff
@@ -66,6 +66,6 @@ def file_test(
             data = data,
             args = args,
             env = cc_env(),
-            local_defines = ["CARBON_FILE_TEST_MANIFEST='\"$(rootpath :{0})\"'".format(tests_file)],
+            local_defines = ["CARBON_FILE_TEST_MANIFEST='\"{0}\"'".format(tests_file)],
```

Comment:
Sorry, I don't think this is going to work in environments that we need to handle. The assumption that the manifest and exe are adjacent is probably not going to be true in `bazel` execution. That's why the rootpath approach: it gives a runfiles-relative path.

Maybe we just need to generate a cpp embed for this. I'll look into that.

Reply:
Okay, thanks, I will close this then